tests/flask/app.py

```diff
@@ -0,0 +1,31 @@
+import strawberry
+from flask import Flask
+from strawberry.file_uploads import Upload
+from strawberry.flask.views import GraphQLView as BaseGraphQLView
+
+
+def create_app(**kwargs):
```

Comment:
can you make this a fixture?

Reply:
I'm going to leave this one as it is since it gets called with a kwarg

Reply:
yes, I didn't notices it is used in multiple places 😊